app/src/main/java/org/coepi/android/worker/tcnfetcher/ContactsFetchManager.kt

```diff
@@ -22,7 +22,9 @@ class ContactsFetchManager(context: Context) {
             .setRequiredNetworkType(CONNECTED)
             .build()
 
-        return Builder(ContactsFetchWorker::class.java, 15, MINUTES)
+        val interval : Long = 15
```

Comment:
Was it needed to declare the type as `Long`? 🤔

Reply:
Not declaring it was causing a lint error.

`Errors found:
  
  /home/runner/work/app-android/app-android/app/src/main/java/org/coepi/android/worker/tcnfetcher/ContactsFetchManager.kt: Error: Unexpected failure during lint analysis of ContactsFetchManager.kt (this is a bug in lint or one of the libraries it depends on)
  
  Message: java.lang.Integer cannot be cast to java.lang.Long
  
  The crash seems to involve the detector androidx.work.lint.InvalidPeriodicWorkRequestIntervalDetector.
  You can try disabling it with something like this:
      android {
          lintOptions {
              disable "InvalidPeriodicWorkRequestInterval"
          }
      }`

Reply:
Okay. Did you try adding `L` at the end of the number?

Reply:
I did not, fixed.